web/src/main/webapp/css/classic/main.css

```diff
@@ -11,7 +11,6 @@
 
 BODY {
 	font-family: "Trebuchet MS", Arial, Helvetica, sans-serif;
-	font-size: 75%;
```

Comment:
That actually looks pretty good on lower resolutions. Still, the icon vertical alignment is an issue.